src/libaktualizr/uptane/fetcher_test.cc

```diff
@@ -81,9 +89,10 @@ void test_pause(const Uptane::Target& target) {
   std::promise<void> end_pausing;
 
   EXPECT_EQ(f.setPause(true), PauseResult::kNotDownloading);
+  EXPECT_EQ(f.setPause(false), PauseResult::kNotPaused);
 
   std::thread([&f, &end_pausing] {
-    std::this_thread::sleep_for(std::chrono::milliseconds(200));
+    std::this_thread::sleep_for(std::chrono::milliseconds(400));  // wait for download start
```

Comment:
Could we change this to something that waits only as long as necessary? I think we should move away from waiting an arbitrary number of seconds or milliseconds.